components/list/list.css

```diff
@@ -8,4 +8,14 @@
   list-style-type: upper-roman;
 }
 
+ul.stacked-list {
+  padding-left: 0;
+  list-style: none;
+  padding-inline-start: 0;
```

Comment:
padding - inline - start and padding left are same properties, so avoid writing same code and remove one of them

Reply:
Yes +1 to this you can remove one of them
it would be better if you keep padding-inline-start:0 and remove padding-left
padding-inline-start gives better accessibility :-> 
Even I use it like this way inline-start and block-start

Reply:
thanks for pointing this put guys.